src/App.js

```diff
@@ -11,18 +12,19 @@ class App extends Component {
     super();
     this.state = {
       movies: [],
-      selectedMovie:'',
-      error: ''
+      selectedMovie: null,
+      error: null
       //if there is a ChosenOne - get all the info on that movie
       //if not, then have all the information
     }
   }
 
   // componentWillMount() {
   componentDidMount() {
-    const url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies';
-    fetch(url)
-    .then(res => res.json())
+    getAllMovies()
+    // const url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies';
+    // fetch(url)
+    // .then(res => res.json())
```

Comment:
Why keep this code as comments? I would remove it. Remember - you can always access your old code by looking at previous commits and/or branches.